packages/amplify_authenticator/example/README.md

```diff
@@ -0,0 +1,16 @@
+# amplify_authenticator_example
+
+A new Flutter project.
```

Comment:
Now would be a good time to update this file, since it would be easy to forget later.

Reply:
working on it